src/main/java/hudson/plugins/git/util/AncestryBuildChooser.java

```diff
@@ -55,50 +55,48 @@ public String getAncestorCommitSha1() {
         
         // filter candidates based on branch age and ancestry
-        return git.withRepository(new RepositoryCallback<List<Revision>>() {
-            public List<Revision> invoke(Repository repository, VirtualChannel channel) throws IOException {
-                try (RevWalk walk = new RevWalk(repository)) {
-
-                    RevCommit ancestor = null;
-                    if (!Strings.isNullOrEmpty(ancestorCommitSha1)) {
-                        try {
-                            ancestor = walk.parseCommit(ObjectId.fromString(ancestorCommitSha1));
-                        } catch (IllegalArgumentException e) {
-                            throw new GitException(e);
-                        }
-                    }
+        return git.withRepository((Repository repository, VirtualChannel channel) -> {
+            try (RevWalk walk = new RevWalk(repository)) {
 
-                    final CommitAgeFilter ageFilter = new CommitAgeFilter(maximumAgeInDays);
-                    final AncestryFilter ancestryFilter = new AncestryFilter(walk, ancestor);
+                RevCommit ancestor = null;
+                if (!Strings.isNullOrEmpty(ancestorCommitSha1)) {
+                    try {
+                        ancestor = walk.parseCommit(ObjectId.fromString(ancestorCommitSha1));
+                    } catch (IllegalArgumentException e) {
+                        throw new GitException(e);
+                    }
+                }
 
-                    final List<Revision> filteredCandidates = Lists.newArrayList();
+                final CommitAgeFilter ageFilter = new CommitAgeFilter(maximumAgeInDays);
+                final AncestryFilter ancestryFilter = new AncestryFilter(walk, ancestor);
 
-                    try {
-                        for (Revision currentRevision : candidates) {
-                            RevCommit currentRev = walk.parseCommit(ObjectId.fromString(currentRevision.getSha1String()));
+                final List<Revision> filteredCandidates = Lists.newArrayList();
 
-                            if (ageFilter.isEnabled() && !ageFilter.apply(currentRev)) {
-                                continue;
-                            }
+                try {
+                    for (Revision currentRevision : candidates) {
+                        RevCommit currentRev = walk.parseCommit(ObjectId.fromString(currentRevision.getSha1String()));
 
-                            if (ancestryFilter.isEnabled() && !ancestryFilter.apply(currentRev)) {
-                                continue;
-                            }
+                        if (ageFilter.isEnabled() && !ageFilter.apply(currentRev)) {
+                            continue;
+                        }
 
-                            filteredCandidates.add(currentRevision);
+                        if (ancestryFilter.isEnabled() && !ancestryFilter.apply(currentRev)) {
```

Comment:
BTW [whitespace-reduced diff](https://github.com/jenkinsci/git-plugin/pull/565/files?w=1)

Reply:
I assume this does not require a change.  The new indentation is correct (as was the old).  The whitespace reduced diff is a nice way to see that the key change was the switch to use a lambda.